### ISSUE TITLE
Create 'fluo remove' command

### ISSUE THREAD

**Issue by mikewalch** [apache/fluo#991]
After a Fluo application is initialized using `fluo init`, metadata exists for the application in Zookeeper.  Currently, there is no way to remove this metadata without using the Zookeeper CLI   It would be good to have a `fluo remove` command that removes this metadata.

**Comment by alanblueshift** [comment]
I started to implement this doing the same thing the _init -f_ does but without the init part. 
But I was wondering what happen if I remove the tables and the zookeeper data without stop the oracle and the workers before. Maybe apply the same validation of the _init_ command:

```java
if (admin.applicationRunning()) {
    System.err.println("Error - The Fluo '" + config.getApplicationName() + "' application"
        + " is already running and must be stopped before running 'fluo init'. "
        + " Aborted initialization.");
    System.exit(-1);
}
```

OR we could store the PID of the workers and oracle in a file like .[APP_NAME]_**woker01**.pid, .[APP_NAME]_**worker02**.pid, APP_NAME]_**oracle01**.pid.... and when a remove command it is called if a **--force** flag we could find and kill this processes too.

And... If we have the PID files we could implement commands like:
```shell
fluo worker -a myapp -k
fluo worker -a myapp -k
```
AND 
```shell
fluo worker -a myapp --kill
fluo worker -a myapp --kill
```

**Comment by mikewalch** [comment]
I think the first option is the best. You should check to see if the application is running and abort if it is. I was thinking the `remove` command was only run once from one node (like the `init` command). It just cleans up metadata in Zookeeper and drops the Accumulo table.  Stopping processes is tricky as they could be running locally, in YARN, Marathon, or Kubernetes.

**Comment by kpm1985** [comment]
This issue is the same as #529 from what I can tell.

**Comment by kpm1985** [comment]
Can we close this? Can we close #529?

**Comment by mikewalch** [comment]
Closed by #999
